docs/src/modules/components/MarkdownElement.js

```diff
@@ -379,7 +379,7 @@ const Root = styled('div')(
         },
       },
     },
-    '& a, & a code': {
+    '& a, & a code, & strong': {
```

Comment:
This doesn't work, e.g. https://mui.com/careers/react-engineer-x-charts/

before
<img width="705" alt="Screenshot 2023-10-11 at 00 52 54" src="https://github.com/mui/material-ui/assets/3165635/98735dd9-3280-4dcb-be09-31952eaab969">

after

<img width="697" alt="Screenshot 2023-10-11 at 00 52 20" src="https://github.com/mui/material-ui/assets/3165635/5ca5a1fd-07c9-4236-9a59-502e9a1ab171">

Reply:
Fixed in #39384.